src/measure_theory/Meas.lean

```diff
@@ -24,6 +27,34 @@ def of (X : Type u) [measurable_space X] : Meas := ⟨X⟩
 -- -- If `measurable` were a class, we would summon instances:
 -- local attribute [class] measurable
 -- instance {X Y : Meas} (f : X ⟶ Y) : measurable (f : X → Y) := f.2
+
+def Measure : Meas ⥤ Meas :=
+⟨ λX, ⟨@measure_theory.measure X.1 X.2⟩,
```

Comment:
I personally find the "named constructor"-version slightly more readable than the anonymous version.

Reply:
True, but there are two problems:
1. Lean was crashing when writing down the structure syntax (some assertions), and 
2. the auto-parameters took ages to fail

I can use the structure syntax again.